src/boot/idle.c

```diff
@@ -77,14 +77,38 @@ void Idle_ThreadEntry(void* arg) {
             gViConfigMode = osViModeMpalLan1;
             break;
 
-#if OOT_DEBUG
         case OS_TV_PAL:
             gViConfigModeType = OS_VI_FPAL_LAN1;
             gViConfigMode = osViModeFpalLan1;
             gViConfigYScale = 0.833f;
             break;
-#endif
     }
+#elif !OOT_PAL_N64
```

Comment:
haven't been paying attention to how we've been handling versioning flags, but to me it seems more sensible to invert and write out the OOT_PAL_N64 case first.

Reply:
there's some discussion about it here: https://discord.com/channels/688807550715560050/688851317593997489/1283275555822768138

basically the idea is to always put NTSC first, but I agree it might look weird